daylight_saving.md

```diff
@@ -0,0 +1,59 @@
+NTFS Timezones and Daylight saving
+=================
+
+## Handling time-zones
+
+The `network_timezone` field in file networks.txt sets the time zone for all `stop_times` of a `trip` in the `network`. 
+
+If a feed covers multiple time zones, a specific time zone information can be specified on a `stop_point` in the `stop_timezone` field. Set this optional field in file `stops.txt` (for stops with location_type = 0). If not set, the timezone of the network is used. Ensure that trip travel times remain accurate and consistent by providing departure and arrival times based on the `network_timezone` for stops that are out of that timezone.
+
+Navitia presents time for departure and arrival times in the local time zone of each stop. \
+Warnings: 
+- Time in `stop_times.txt` should always increase along a trip. 
+- NTFS uses the local timezone at noon of the travel day of the trip, following the same rule as (GTFS Time data format)[http://gtfs.org/schedule/reference/#field-types]:
+> Time in the HH:MM:SS format (H:MM:SS is also accepted). The time is measured from "noon minus 12h" of the service day (effectively midnight except for days on which daylight savings time changes occur). For times occurring after midnight, enter the time as a value greater than 24:00:00 in HH:MM:SS local time for the day on which the trip schedule begins. \
+Example: 14:30:00 for 2:30PM or 25:35:00 for 1:35AM on the next day.
+
```

Comment:
I think it would be great to have a documented upper bound
```suggestion
The maximum accepted value is 48:00:00
```

Reply:
If I remember correctly, there are some trains that runs more than 2 days. Is  limit of 48:00 a technical limit ?